gr-qtgui/lib/eye_sink_f_impl.cc

```diff
@@ -0,0 +1,661 @@
+}
+
+std::string eye_sink_f_impl::title() { return d_main_gui->title().toStdString(); }
+
+std::string eye_sink_f_impl::line_label(unsigned int which)
+{
+    return d_main_gui->lineLabel(0).toStdString();
+}
+
+std::string eye_sink_f_impl::line_color(unsigned int which)
+{
+    return d_main_gui->lineColor(0).toStdString();
+}
+
+int eye_sink_f_impl::line_width(unsigned int which) { return d_main_gui->lineWidth(0); }
+
+int eye_sink_f_impl::line_style(unsigned int which) { return d_main_gui->lineStyle(0); }
+
+int eye_sink_f_impl::line_marker(unsigned int which) { return d_main_gui->lineMarker(0); }
+
+double eye_sink_f_impl::line_alpha(unsigned int which)
+{
+    return (double)(d_main_gui->markerAlpha(0)) / 255.0;
+}
+
+void eye_sink_f_impl::set_nsamps(const int newsize)
+{
+    if (newsize != d_size) {
+        gr::thread::scoped_lock lock(d_setlock);
+
+        // Set new size and reset buffer index
+        // (throws away any currently held data, but who cares?)
+        d_size = newsize;
+        d_buffer_size = 2 * d_size;
+        int d_sps = d_main_gui->getSamplesPerSymbol();
+
+        // Resize buffers and replace data
+        for (unsigned int n = 0; n < d_nconnections + 1; n++) {
+            volk_free(d_buffers[n]);
+            d_buffers[n] = (double*)volk_malloc(d_buffer_size * sizeof(double),
+                                                volk_get_alignment());
+            memset(d_buffers[n], 0, d_buffer_size * sizeof(double));
+
+            volk_free(d_fbuffers[n]);
+            d_fbuffers[n] =
+                (float*)volk_malloc(d_buffer_size * sizeof(float), volk_get_alignment());
+            memset(d_fbuffers[n], 0, d_buffer_size * sizeof(float));
+        }
+
+        // If delay was set beyond the new boundary, pull it back.
+        if (d_trigger_delay > 2 * d_sps) {
+            GR_LOG_WARN(d_logger,
+                        boost::format("Trigger delay (%1%) outside of display range "
+                                      "(0:%2%). Moving to 50%% point.") %
+                            (d_trigger_delay / d_samp_rate) %
+                            ((2 * d_sps) / d_samp_rate));
+            d_trigger_delay = d_sps;
+            d_main_gui->setTriggerDelay(d_trigger_delay / d_samp_rate);
+        }
+
+        d_main_gui->setNPoints(d_size);
+        _reset();
+    }
+}
+
+void eye_sink_f_impl::set_samp_rate(const double samp_rate)
+{
+    gr::thread::scoped_lock lock(d_setlock);
+    d_samp_rate = samp_rate;
+    d_main_gui->setSampleRate(d_samp_rate);
+}
+
+int eye_sink_f_impl::nsamps() const { return d_size; }
+
+void eye_sink_f_impl::enable_stem_plot(bool en)
+{ /* Unused, but do not remove*/
+}
+
+void eye_sink_f_impl::enable_menu(bool en) { d_main_gui->enableMenu(en); }
+
+void eye_sink_f_impl::enable_grid(bool en) { d_main_gui->setGrid(en); }
+
+void eye_sink_f_impl::enable_autoscale(bool en) { d_main_gui->autoScale(en); }
+
+void eye_sink_f_impl::enable_semilogx(bool en)
+{ /* Unused, but do not remove*/
```

Comment:
Please update this and the other unused function calls that are just comments to have the comment on the following line and indented.